document/js-api/index.bs

```diff
@@ -267,6 +267,8 @@ namespace WebAssembly {
 
     Promise&lt;Instance> instantiate(
         Module moduleObject, optional object importObject);
+
+    readonly attribute Tag JSTag;
```

Comment:
This might not actually be needed given the algorithm below.

Reply:
If the attribute works (and I don't see why it wouldn't), we should prefer it over further manual adjustments to the "create a namespace object" algorithm. You'll need to add the getter steps, but those should be trivial.

Reply:
Done.